src/checkout/sections/Summary/Summary.tsx

```diff
@@ -39,7 +40,12 @@ export const Summary: FC<SummaryProps> = ({
 	discount,
 }) => {
 	return (
-		<div className="flex h-fit w-full flex-col overflow-hidden rounded lg:overflow-visible">
+		<div
+			className={clsx(
+				"z-0 flex h-fit w-full flex-col overflow-hidden rounded lg:overflow-visible",
```

Comment:
I believe setting the overflow here doesn't do much? So we can skip both `overflow-hidden` and `overflow-visible`